Stack/Opc.Ua.Core/Types/Encoders/JsonDecoder.cs

```diff
@@ -2605,6 +2756,109 @@ public void Pop()
         #endregion
 
         #region Private Methods
+        private ushort ToNamespaceIndex(string uri)
+        {
+            var index = m_context.NamespaceUris.GetIndex(uri);
+
+            if (index < 0)
+            {
+                if (!UpdateNamespaceTable)
+                {
+                    return UInt16.MaxValue;
```

Comment:
Wouldn't it be better to use -1 or some other negative number to transport this information?

Reply:
namespaceindex is unsigned so MaxValue is a reasonable choice here